### PR TITLE
Update list.md

### PR DESCRIPTION
**Pull request by fau-university-press**
Added Universitätsbibliothek Erlangen-Nürnberg and project Open Source Academic Publishing Suite

### CODE REVIEW

File: list.md [hunk 1]
```diff
@@ -214,6 +214,12 @@ ETH Library Lab, Zürich CH
 Universitätsbibliothek Regensburg
 [https://github.com/ub-regensburg](https://github.com/ub-regensburg)
 
+Universitätsbibliothek Erlangen-Nürnberg
+[https://github.com/UBEN-Webteam](https://github.com/UBEN-Webteam)
+
+project Open Source Academic Publishing Suite (Universitätsbibliothek Erlangen-Nürnberg, Universitäts- und Landesbibliothek Sachsen-Anhalt, SciFlow GmbH)
+[https://gitlab.com/os-aps](https://gitlab.com/os-aps)
+
```
Comment (stweil) [inline list.md:222]: @zuphilip, should we add links to GitLab, too? Or stick to the repository name and only allow bibs on GitHub?

Comment (stweil) [reply list.md:222]: Of course comments from other users are also welcome.

Comment (zuphilip) [reply list.md:222]: Let us discuss this first in a separate issue.

Comment (stweil) [inline list.md:222]:
```
```

Comment (stweil) [inline list.md:222]:
```suggestion
Universitätsbibliothek Erlangen-Nürnberg
[https://github.com/UBEN-Webteam](https://github.com/UBEN-Webteam)

```

**Pull request merged**